src/components/Sidebar.tsx

```diff
@@ -53,8 +56,8 @@ export const Sidebar = () => {
         to="/bookmarks"
         className={({ isActive }) =>
           isActive
-            ? " flex flex-row items-center md:gap-3 gap-1 text-xl px-3 py-2 bg-primaryLight rounded-md dark:bg-darkLight "
-            : " flex flex-row items-center md:gap-3 gap-1 text-xl  px-3 py-2  "
+            ? " flex flex-row items-center md:gap-3 gap-1 px-3 py-2 bg-primaryLight rounded-md dark:bg-darkLight "
+            : " flex flex-row items-center md:gap-3 gap-1 px-3 py-2 hover:bg-primaryLight dark:hover:bg-darkLight rounded-md "
```

Comment:
better to extract this into a separate function as you are using it few more times